integration_tests/src/main/python/string_test.py

```diff
@@ -469,7 +469,29 @@ def test_like_complex_escape():
                 'a like "\\%SystemDrive\\%\\\\\\\\Users%"',
                 'a like "_oo"'),
             conf={'spark.sql.parser.escapedStringLiterals': 'true'})
+
+@pytest.mark.skipif(not is_before_spark_320() or is_databricks91_or_later(), reason='regexp is synonym for RLike starting in Spark 3.2.0')
```

Comment:
This seems to be written backwards and will only run on Spark < 3.2.0 or Databricks 9.1.  Unless Databricks supports these in 9.1, I think this needs to be:
```suggestion
@pytest.mark.skipif(is_before_spark_320(), reason='regexp is synonym for RLike starting in Spark 3.2.0')
```